Sources/Overload/OvRendering/src/OvRendering/Core/ABaseRenderer.cpp

```diff
@@ -58,11 +58,14 @@ void OvRendering::Core::ABaseRenderer::BeginFrame(const Data::FrameDescriptor& p
 	m_basePipelineState = m_driver.CreatePipelineState();
 	SetViewport(0, 0, p_frameDescriptor.renderWidth, p_frameDescriptor.renderHeight);
 
+	OvMaths::FVector4 clearColor = p_frameDescriptor.camera.value().GetClearColor();
+	clearColor.w = 1.0f;
```

Comment:
We should maybe construct the FVector4 directly by parameter to enjoy RVO and avoid the temporary clearColor variable 

`
Clear(
	p_frameDescriptor.camera->GetClearColorBuffer(),
	p_frameDescriptor.camera->GetClearDepthBuffer(),
	p_frameDescriptor.camera->GetClearStencilBuffer(),
	{p_frameDescriptor.camera.value().GetClearColor(), 1.0f}
);
`

Reply:
Yep, that's fair! Good call